perfect-cell.cydsn/atlas_wq_sensor.h

```diff
@@ -30,13 +32,13 @@ struct{
 int atlas_sensor_sleep(uint8 sensor_address);
 
 /**
- * @brief Wake Atlas I2C sensor from sleep mode
+ * @brief Calibrate Atlas sensor using single-point method
  *
  * @param sensor_address I2C slave address of sensor
  *
  * @return 1 on success
  */
-int atlas_sensor_wake(uint8 sensor_address);
+int atlas_sensor_calibrate(uint8 sensor_address);

```

Comment:
`atlas_sensor_sleep` and `atlas_sensor_calibrate` aren't implemented right now. I have them on standby in case we need to calibrate something in the field without unplugging the sensors. For the DO sensor, at least, you can do one-point calibration in the air.